<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>winWebQuery</class>
 <widget class="QMainWindow" name="winWebQuery">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>572</width>
    <height>511</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Query</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 57 12pt &quot;TerminessTTF Nerd Font&quot;;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="cbInterval">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>251</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QDateTimeEdit" name="dteStartDate">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>251</width>
      <height>34</height>
     </rect>
    </property>
    <property name="displayFormat">
     <string>MM/dd/yyyy hh:mm</string>
    </property>
   </widget>
   <widget class="QDateTimeEdit" name="dteEndDate">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>251</width>
      <height>34</height>
     </rect>
    </property>
    <property name="displayFormat">
     <string>MM/dd/yyyy hh:mm</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbDatabase">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>251</width>
      <height>30</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnQuery">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>450</y>
      <width>251</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Query Data</string>
    </property>
    <property name="autoDefault">
     <bool>true</bool>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTextEdit" name="textSDID">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>310</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="tabChangesFocus">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string>SDID</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Database</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Data Interval</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Start Date/Time</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>End Date/Time</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listQueryList">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>30</y>
      <width>251</width>
      <height>281</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAddQuery">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>340</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Query</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>SDID or Site-Method-Parameter</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>0</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Querry List</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRemoveQuery">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>340</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove Query</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSaveQuickLook">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>310</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Save Quick Look</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>380</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Select Quick Look</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbQuickLook">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>410</y>
      <width>251</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLoadQuickLook">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>410</y>
      <width>251</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Load Quick Look</string>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnClearQuery">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>370</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear Query List</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>572</width>
     <height>35</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
